draft-ietf-lamps-csr-attestation.md

```diff
@@ -209,6 +209,45 @@ CSR messages.
 
 # Information Model
 
+## Interaction with an HSM
+
+This specification is intended to be used both in cases where the CSR
+is constructed internally or externally to the attesting environment.
+
+Cases where the CSR is generated internally to the attesting environment
+are straightforward: the generates and embeds Evidence and the corresponding
+certificate chains when constructing the CSR.
+
+Externally-generated CSRs may require extra round-trips of communication
+between the CSR generator and the attesting environment, first to obtain
+the necessary evidence statements about the subject key, and then to use
+the subject key to sign the CSR. For example, consider a CSR generated by
+OpenSSL about a subject key stored on a PKCS#11 device. The interaction would
```

Comment:
```suggestion
a popular crypto library about a subject key stored on a PKCS#11 device. The interaction would
```

Reply:
What reference would you use for PKCS#11?
This one?

   [PKCS11]   RSA Laboratories, "PKCS #11 v2.20: Cryptographic Token
              Interface Standard", Public Key Cryptography Standards
              PKCS#11-v2.20, June 2004.

Reply:
Ugg, yeah, I guess I need to cite PKCS#11. How's this?

  PKCS11:
    author:
      org: OASIS
    title: "PKCS #11 Cryptographic Token Interface Base Specification Version 2.40"
    date: 14 April 2015
    target: http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html